Documented some stuff (#24)
apace100/origins-docs: docs/types/power_types/prevent_sprinting.md

---
title: Prevent Sprinting (Power Type)
date: 2022-06-07
---

# Prevent Sprinting

[Power Type](../power_types.md)

> Prevents the player with this power from sprinting if a condition is met.

Reviewer: ```suggestion
Prevents the player that has the power from sprinting.
```
